README.md

```diff
@@ -48,8 +48,16 @@ end
 doctype html
 html
   head
+    meta name="viewport" content="width=device-width,initial-scale=1.0"
     title This is a title
+    style:
```

Comment:
I'm curious why suddenly we'd do the colon `:` thing here? Is it a limitation of the tokenizer / lexer?

I think `css:` and `javascript:` would be appropriate, but `style` and `script` are real HTML tags that may have attributes and such.

Would it be possible to make them normal tags (no colon)?